docs/TESTS.md

```diff
@@ -1,7 +1,18 @@
-Execute the tests with:
+After configuring `exercism` command-line client with:
 
+    exercism configure --key=YOUR_EXERCISM_KEY
+
+and fetching your first JavaScript exercise with:
+
+    exercism fetch javascript bob
```

Comment:
I worry about duplicating instructions about configuration and fetching (though we probably already do), since if we change it it's going to be pretty hard to remember to get all the other spots.

I do like how it refers to both configuring and fetching, though. That helps orient the person in space and time.